pkg/controller/namespace/deletion/namespaced_resources_deleter.go

```diff
@@ -406,44 +410,46 @@ type gvrDeletionMetadata struct {
 // It returns an estimate of the time remaining before the remaining resources are deleted.
 // If estimate > 0, not all resources are guaranteed to be gone.
 func (d *namespacedResourcesDeleter) deleteAllContentForGroupVersionResource(
+	ctx context.Context,
 	gvr schema.GroupVersionResource, namespace string,
 	namespaceDeletedAt metav1.Time) (gvrDeletionMetadata, error) {
-	klog.V(5).Infof("namespace controller - deleteAllContentForGroupVersionResource - namespace: %s, gvr: %v", namespace, gvr)
+	logger := klog.FromContext(ctx)
+	logger.V(5).Info("Namespace controller - deleteAllContentForGroupVersionResource", "namespace", namespace, "resource", gvr)
 
 	// estimate how long it will take for the resource to be deleted (needed for objects that support graceful delete)
-	estimate, err := d.estimateGracefulTermination(gvr, namespace, namespaceDeletedAt)
+	estimate, err := d.estimateGracefulTermination(ctx, gvr, namespace, namespaceDeletedAt)
 	if err != nil {
-		klog.V(5).Infof("namespace controller - deleteAllContentForGroupVersionResource - unable to estimate - namespace: %s, gvr: %v, err: %v", namespace, gvr, err)
+		logger.V(5).Info("Namespace controller - deleteAllContentForGroupVersionResource - unable to estimate", "namespace", namespace, "resource", gvr, "err", err)
```

Comment:
```suggestion
		logger.V(5).Info("Namespace controller - deleteAllContentForGroupVersionResource - unable to estimate", "namespace", namespace, "groupVersionResource", gvr, "err", err)
```

Reply:
@soltysh Should we change this log to Error instead of Info?

Reply:
Probably not. The error is being handled automatically and not something that an admin needs to deal with.

Reply:
Yeah, we're returning error to the caller, so Info(5) is good enough, if you want to dive deeper.